java/vector/src/main/java/org/apache/arrow/vector/NullableBitVector.java

```diff
@@ -279,6 +279,8 @@ public Boolean getObject(int index) {
    public void copyFrom(int fromIndex, int thisIndex, NullableBitVector from) {
       if (from.isSet(fromIndex) != 0) {
          set(thisIndex, from.get(fromIndex));
+      } else {
+         BitVectorHelper.setValidityBit(validityBuffer, thisIndex, 0);
```

Comment:
Do we have a unit test for this?

Reply:
We have some copy() related unit tests but in any case, I am planning to write more unit tests in the next couple of weeks to get overall good coverage. I don't think so I can get to them now because of time constraints. 

Note that behavior of copyFrom() hasn't changed.

Reply:
Ok